application/src/main/java/org/togetherjava/tjbot/commands/utils/CandidateScore.java

```diff
@@ -0,0 +1,8 @@
+package org.togetherjava.tjbot.commands.utils;
+
+record CandidateScore(String candidate, int score) implements Comparable<CandidateScore> {
```

Comment:
This class is obsolete when u use the comparator-api (as shown in my rework example)